easypost/requestor.py

```diff
@@ -91,7 +116,11 @@ def request_raw(
                 f"at {SUPPORT_EMAIL} for assistance."
             )
 
-        abs_url = "%s%s" % (api_base, url or "")
+        if beta:
+            abs_url = "%s%s" % ("https://api.easypost.com/beta", url or "")
```

Comment:
Please use `f strings` in Python, it's much easier to read. This approach you are using here was prevalent in Python2 but not so much anymore due to the introduction of `f strings`.

Reply:
resolved